utopia-api/src/property-controls/property-controls.ts

```diff
@@ -216,6 +258,21 @@ export type RegularControlDescription = BaseControlDescription | HigherLevelCont
 // with any changes to this or the component types.
 export type ControlDescription = RegularControlDescription | FolderControlDescription
 
+type ControlBaseFields = {
+  control: RegularControlType
+  label?: string
+  defaultValue?: unknown
+  visibleByDefault?: boolean
+}
+function getControlSharedFields(control: RegularControlDescription): ControlBaseFields {
+  return {
+    control: control.control,
+    label: control.label,
+    defaultValue: control.defaultValue,
+    visibleByDefault: control.visibleByDefault,
+  }
+}
+
```

Comment:
I think this should either live at the very top or very bottom of the file, with a comment explaining its purpose.

Reply:
I moved it to the top.